pkg/ccl/backupccl/restore_span_covering.go

```diff
@@ -64,6 +64,17 @@ var targetOnlineRestoreSpanSize = settings.RegisterByteSizeSetting(
 	16<<30,
 )
 
+var maxFileCount = settings.RegisterIntSetting(
+	settings.SystemOnly,
+	"backup.restore_span.max_file_count",
+	"the max number of backup files an extending restore span may contain",
```

Comment:
nit s/max/maximum/ in the description. Also I'm not sure "an extending restore span" is going to be clear to a user. But if we don't make this public I mind less.

Reply:
> nit s/max/maximum/ in the description
Done

Didn't bother with further changes as this setting is private.